tools/oeedger8r/intel/Lexer.mll

```diff
@@ -88,7 +88,7 @@ rule tokenize = parse
   | "import"     { Timport }
   | "allow"      { Tallow }
   | "public"     { Tpublic }
-  | "transition_using_threads"       { Tswitchless }
+  | "switchless"       { Tswitchless }
```

Comment:
Hm...why are we diverging from Intel's EDL here?

Reply:
There was a discussion during the design review meeting. People doesn't like the term `transition_using_threads`. To me personally, the term is confusing and too verbose. We think `switchless` is a more intuitive term.

Do we need to follow Intel's EDL literally?

Reply:
Based on the design review, lets go with "switchless". Thanks

Reply:
@radhikaj Who all agreed on that? I think that design review happened while I was out of town for the FCA hackathon.

I would _highly recommend_ we not diverge from Intel's syntax, as our original intent was to be as compatible with Intel EDL as possible. This is an unnecessary divergence that's being made just for cosmetic reasons, so it doesn't seem worth it to me.

@mikbras were you able to attend that design meeting? I am looking for minutes right now...

Reply:
@andschwa Mike is on vacation. Let's continue the discussion through emails.

Reply:
> @radhikaj Who all agreed on that? I think that design review happened while I was out of town for the FCA hackathon.
> 
> I would _highly recommend_ we not diverge from Intel's syntax, as our original intent was to be as compatible with Intel EDL as possible. This is an unnecessary divergence that's being made just for cosmetic reasons, so it doesn't seem worth it to me.
> 
> @mikbras were you able to attend that design meeting? I am looking for minutes right now...

@andschwa Attendees of the meeting agreed to it. @mikbras was in the meeting as well. Thanks @jxyang for starting a thread.

Reply:
FWIW, I have no idea what "switchless" means and so don't think that term is intuitive.